sherpa/astro/data.py

```diff
@@ -283,15 +283,16 @@ def _extract_fields(obj, summary):
 def html_pha(pha):
     """HTML representation: PHA"""
 
-    from sherpa.astro.plot import DataPHAPlot, backend
+    from sherpa.astro.plot import DataPHAPlot
+    from sherpa import plot
 
     ls = []
 
     plotter = DataPHAPlot()
```

Comment:
I guess it's okay to have

    plotter = DataPHAPlot()

here rather than an explicit symbol like `sherpa.astro.plot.DataPHAPlot` since we've just imported the symbol in this routine (I don't know the rules to know if Python caches this or will re-import the symbols)?

I would like to think the tests would catch this (if it wasn't working) but things are a bit squirrely so I'm not 100% sure of it.

Reply:
I'm not 100% sure either. I'll have a look. However, I find that testing this in a good way is hard without a second functioning backend. Sure, I can use the Dummy backend, but if I don't see anything, I'm not quite sure if that's because it silently failed or because it (correctly) did not produce a plot. So, while I'm going to look into this a little more here, there might be a few cases like that that are not fully resolved here, but that certainly will pop up when the full bokeh backend is merged in and tested. 

The important point is that this is not doing any harm. It worked before with the pylab backend, and it's going to continue to work with one backend in the future. The only thing that might happen is that this PR does not fully live up to it's promise - elimination 100% of the imports that hinder backend-switching. BUT, that's not introducing any new problems, it's just "not fixing all existing problems". If I can't figure that out here, I'll add an issue to make sure I'll get back to it when I write/update the step 1 code.

Reply:
I am wondering if there's a step between stages 0 and 1 in your master plan to stop and review the plotting tests, perhaps re-arranging some of them and looking at the backend-switcher stuff?

Reply:
The testing will be in Step 1 A (turn backends into classes). Step 1 B will be "add bokek class".

Reply:
Yes, I thought about this more. It is OK to import `DataPHAPlot` because that is a Sherpa plot object, not a backend. The Plot objects are (or at least will be) backend-indenpendent. The change here is that it previously also imported "backend" which we don't want. It should use "plot.backend" instead.